src/pages/students/Students.tsx

```diff
@@ -85,7 +85,15 @@ function Students() {
       <div className="form-group">
         <div className="column1">
           <label>DNI:</label>
-          <input {...register("dni")} type="number" />
+          <input
+            {...register("dni")}
+            type="text"
+            onKeyDown={(e) => {
+              if (e.key === "." || e.key === ",") {
+                e.preventDefault();
+              }
```

Comment:
En este habría que agregar un par de caracteres que el number te deja escribir


```suggestion
              if (e.key === "." || e.key === "," || e.key === "-" || e.key === "+") {
                e.preventDefault();
              }
```